akka-projection-cassandra/src/test/resources/reference.conf

```diff
@@ -0,0 +1,2 @@
+# To use docker-compose instead of testcontainers comment out this configuration
+akka.projection.cassandra.session-provider = "akka.projection.cassandra.ContainerSessionProvider"
```

Comment:
overriding reference.conf isn't supported (classpath ordering).

I prefer to keep test configuration in the tests (in code):
```
class CassandraProjectionSpec extends ScalaTestWithActorTestKit("""
  akka.projection.cassandra.session-provider = "akka.projection.cassandra.ContainerSessionProvider"
  """)
```

Reply:
I didn't know that, somehow it still merged it in and worked though.

I removed the `reference.conf` and added a shared config in the Cassandra tests. Does that work or would you prefer it be repeated per spec?

Reply:
classpath order is undefined, although I think sbt will always put resources in same project first